src/main.js

```diff
@@ -606,6 +660,24 @@
         });
       })
     }
+    checkForParametersInConfig(config, ...params){
+      for (let c = 0; c < params.length; c++){
+        if(!(params[c] in config) || config[params[c]] === undefined){
+           Log.error(`No paramenter added! Please provide ${params[c]} parameter.`);
+           return false;
+        }
+      }
+    }
+
+    checkPath(...params){
+      const fs = require('fs');
+      for (let c = 0; c < params.length; c++){
+        if (!fs.existsSync(params[c])) {
+            Log.error(`${params[c]} param doesn\`t exist! Please provide existing path.`);
```

Comment:
Fix the message " folder does not exist. Use --force to create it"
Add folder creation on --force